src/DevChatter.DevStreams.Core/CollectionExtension.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DevChatter.DevStreams.Core
+{
+    public static class CollectionExtension
+    {
+        private static Random _random = new Random();
+
+        /// <summary>
+        /// Pull random items from set
```

Comment:
Should mention that this is pulling a sequence of items, not independently random ones.